README.md

```diff
@@ -198,9 +233,11 @@ If you want multiple test users, it's recommended to include their credentials i
 cy.logout();
 ```
 
-| Property | Type | Default value | Required? |
-|------|------|------|------|
-| `returnTo` | `String` | None | No |
+| Property            | Type     | Default value | Required? |
+| ------------------- | -------- | ------------- | --------- |
+| `options`           | `Object` | None          | No        |
+| `options.returnTo`  | `String` | None          | No        |
+| `options.logoutUrl` | `String` | None          | No        |
```

Comment:
I appreciate you making sure the documentation stays up-to-date with the feature addition - this is so important!

Reply:
Thank you, yeah it is!